server/web/session.go

```diff
@@ -67,6 +67,16 @@ func loginCheckMiddleware(in echo.HandlerFunc) echo.HandlerFunc {
 		if isLoggedIn {
 			return in(c)
 		}
-		return c.NoContent(http.StatusForbidden)
+
+		errorMessage := "Access restricted, please log in first"
```

Comment:
Sorry for the nit :eyes: ... but could you please call this `msg` .. (or simply`) it does sound more idiomatic go.